R/vst02.R

```diff
@@ -147,5 +148,6 @@ vst02_2_main <- modify_default_args(vst02_1_main, exclude_base_abn = TRUE)
 vst02_2 <- chevron_t(
   main = vst02_2_main,
   preprocess = vst02_pre,
-  postprocess = vst02_post
+  postprocess = vst02_post,
+  dataset = c("adsl", "advs")
```

Comment:
the order of this is different from the rest, I actually think adsl should always come first, but one may argue, that adsl is only used for the total patient count, not the main dataset, nonetheless, should keep them consistent

Reply:
let me put it first for now